apps/server/src/routes/swagger/schemas.yml

```diff
@@ -66,62 +66,74 @@ components:
-        - entities
-        - state
-        - createdAt
-        - createdBy
       properties:
         id:
-          type: string
-          description: the id of the registration
-        categoryId:
-          type: string
-          description: the program short name this registration is for
-        entities:
-          type: object
-          description: submitted data
-          properties:
-            entityType:
-              $ref: '#/components/schemas/Entities'
+          type: number
+          description: Id of the Submission
         state:
           type: string
-          description: the state of the submission
+          description: State of the Submission
           enum: ['OPEN', 'VALID', 'INVALID', 'CLOSED', 'COMMITED']
-        createdBy:
+        organization:
           type: string
-          description: the user who created the submission
+          description: Organization the Submission belongs to
+        data:
+          type: object
+          additionalProperties:
+            type: object
+            properties:
+              creator:
+                type: string
+                description: The user who created the submission
+              records:
+                type: array
+                items:
+                  type: object
+                  description: Content of the file in JSON format
+              batchName:
+                type: string
+                description: Original filename  of the submission
+              dataErrors:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ValidationError'
+        errors:
+          type: object
+          additionalProperties:
+            type: array
+            items:
+              $ref: '#/components/schemas/ValidationError'
         createdAt:
           type: string
-          description: the date/time of the creation
-    Entities:
-      type: array
-      items:
-        $ref: '#/components/schemas/SubmissionEntity'
-    SubmissionEntity:
-      required:
-        - batchName
-        - creator
-        - records
-        - dataErrors
-      properties:
-        batchName:
+          description: The date/time of the creation
+        createdBy:
           type: string
-          description: the filename must start with an entity type (e.g. donor, sample) and have .tsv extension
-        creator:
+          description: The user who created the submission
+        udpatedAt:
           type: string
-          description: first name and last name of the user who add this entity data
-        records:
-          type: array
-          description: content of the file in JSON format
-        dataErrors:
-          type: array
-          items:
-            $ref: '#/components/schemas/ValidationError'
+          description: The date/time of the last update
+        updatedBy:
+          type: string
+          description: The user who last updated the submission
```

Comment:
User name or ID? Could be more explicit here.

Reply:
Adding `user name` in the description as it is more useful